components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2899,53 +2897,61 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
                 });
             });
         } catch (err) {
-            if (AuthorizerError.is(err)) {
-                await this.authorizer.writeRelationships(removeUserFromOrg(team.id, userId));
-            }
+            await this.authorizer.writeRelationships(removeUserFromOrg(team.id, userId));
 
             throw err;
         }
     }
 
-    public async removeTeamMember(ctx: TraceContext, teamId: string, userId: string): Promise<void> {
-        traceAPIParams(ctx, { teamId, userId });
+    public async removeTeamMember(ctx: TraceContext, orgID: string, userId: string): Promise<void> {
+        traceAPIParams(ctx, { teamId: orgID, userId });
 
         if (!uuidValidate(userId)) {
             throw new ResponseError(ErrorCodes.BAD_REQUEST, "user ID must be a valid UUID");
         }
 
-        const currentUser = await this.checkAndBlockUser("removeTeamMember");
+        const requestor = await this.checkAndBlockUser("removeTeamMember");
```

Comment:
Renamed to make it a bit clearer that it is the person asking for someone to be removed, rather than the `currentUser` to be removed